docs/user-guide/ze-create-zowe-explorer-cics-ext-profiles.md

```diff
@@ -1,99 +1,77 @@
+# Managing Zowe Explorer CICS Extension profiles
 
-You need to have a Zowe Explorer profile to use all functions of the extension.
+To use all functions of the Zowe Explorer CICS extension, you need to have a Zowe Explorer profile.
 
-If you already have a Zowe CICS CLI profile, the CICS tree loads the default profile on startup.  
+If you already have a Zowe CICS CLI profile, the CICS tree in the VS Code **Side Bar** loads the default profile on startup.  
 
-If you do not have an existing Zowe CICS CLI profile, follow these steps to create one:
+If you do not have an existing Zowe CICS CLI profile, refer to the following instructions.
 
 ## Using Zowe team configuration
 
-1. Select the **+** button in the CICS tree.
+1. Expand the **CICS** tree in the **Side Bar**, and click the **+** icon.
 
-2. Select the **Create New CICS profile** option to open the config file.
+2. In the **Quick Pick** menu, select the **Create New CICS profile** option to open the configuration file.
 
-3. Edit the config file to add a CICS profile.
+   The associated **[is there a better way/word to describe this?]** configuration file opens in an **Editor**.
 
-4. Save the config file.
+3. Edit the configuration file to add a CICS profile.
 
-5. Click the **Refresh** button at the top level of the CICS tree or run the `Zowe Explorer for IBM CICS: Refresh` option in the command palette to refresh the Zowe Explorer for IBM CICS extension.
+4. Save the configuration file.
 
-6. Select the **+** button in the CICS tree and click the newly created profile to load it into view.
+5. Refresh the Zowe Explorer for IBM CICS extension. Either click the **Refresh** icon at the top of the **CICS** tree in the **Side Bar**, or select the `Zowe Explorer for IBM CICS: Refresh` option in the **Command Palette**.
 
-![Zowe CICS Explorer profiles from config](../images/ze-cics/create-profile-from-config.gif)
+6. In the **CICS** tree, select the **+** icon and select the newly created profile from the **Quick Pick** menu to display it in the **CICS** tree.
 
-### Using Zowe V1 profiles
+   ![Zowe CICS Explorer profiles from config](../images/ze-cics/create-profile-from-config.gif)
 
-1. Select the **+** button in the CICS tree.
+::: Note
 
-2. Select the **Create a CICS profile** option to open a panel that defines connection details.
-
-**Note**: The connection must point to a CICS region's CICS Management Client Interface (CMCI) TCP/IP host name and port number. The region can be a WUI server in a CICSPlex, or else a stand-alone Single Management Application Programming (SMSS) region.  
+The connection **[To what? between what two things?]** must point to a CICS region's CICS Management Client Interface (CMCI) TCP/IP host name and port number. The region can be a WUI server in a CICSPlex, or a stand-alone Single Management Application Programming (SMSS) **[is SMSS really the correct acronym? the letters don't match the extended name]** region.  
```

Comment:
@adam-wolfe - please respond

Reply:
Rather than "The connection must point..." I would say "The CICS profile must specify a CICS region's ..."

I would change the second sentence to "The region can be a WUI server in a CICSPlex or a stand-alone CICS System Management Single Server (SMSS) region."